test/evergreen.yml

```diff
@@ -1501,6 +1501,69 @@ tasks:
         vars:
           checkpoint_args: -t c -n 1000000 -k 5000000 -C cache_size=100MB
 
+  - name: coverage-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/gcc CXX=/opt/mongodbtoolchain/v3/bin/g++ PATH=/opt/mongodbtoolchain/v3/bin:$PATH CFLAGS="--coverage -fPIC -ggdb" LDFLAGS=--coverage
+          posix_configure_flags: --enable-silent-rules --enable-diagnostic --enable-strict --enable-python --with-builtins=lz4,snappy,zlib
+      - func: "make check all"
+      - func: "unit test"
+        vars:
+          unit_test_args: -v 2 --long
+      - func: "test format"
+        vars:
+          test_format_args: -1 -q -c ../../../test/format/CONFIG.stress checkpoints=1 leak_memory=0 mmap=1 file_type=row compression=snappy logging=1 logging_compression=snappy logging_prealloc=1
```

Comment:
There seem to be lots of duplicated format arg settings, and `-1 -q -c ../../../test/format/CONFIG.stress checkpoints=1 leak_memory=0 mmap=1` is pretty standard in this test. Can we split the `test_format_args` into multiple sub args and give them a reasonable default value, so that we can save some settings here?

Reply:
Well, I'm not sure about this. I want to minimize the number of arguments we introduce. I don't want to go into a path that we can potentially create argument for each command line options.

Reply:
Probably too expensive for each option, but might be worthwhile for some frequently used ones e.g. `-c <cfg_file>`, `-1 (times to run)`, `file_type` etc. 
I'm not having a strong opinion though, just wondering what we can do to reduce duplication.